src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/CloudException.cs

```diff
@@ -24,7 +29,21 @@ public class CloudException : RestException
         /// <summary>
         /// Gets or sets the response object.
         /// </summary>
-        public CloudError Body { get; set; }
+        public CloudError Body
```

Comment:
Can we just leave the property as `Body` and not bother introducing an `ErrorBody` member?

Reply:
Some Thoughts:
- The class `RestExceptionGeneric` holds the deserialized model object into `ErrorBody` which also makes semantic sense.
- The reason I introduced `ErrorBody` property was to bring this consistency which will further allow us to effectively use the generic methods in out auto-generated client methods.
- We may perhaps flip it and have the generic exception class and put the `Body` property in it instead to enforce this consistency, however we risk running into conflicting property names since `Body` is overly generic.

Reply:
Yeah, I get that the e deserialized model object is in `ErrorBody` in the `RestExceptionGeneric` -- and I'll grant that it's not really specific just calling it `Body`, but given the existing property name, I'd opt to go the other way and change `RestExceptionGeneric`'s `ErrorBody` to `Body` and make it `Body` everywhere.